# although openni is started with registered:=true  there is no data arriving at
# /camera/depth_registered/image. Therefore, we use /camera/depth
depth_image_topic: /camera/depth/image_rect_raw
camera_info_topic: /camera/depth/camera_info
downsampling_factor: 8

resolution:
  width: 640
  height: 480

